spec/defines/foreman_rake_spec.rb

```diff
@@ -23,7 +23,7 @@
           .with_user('foreman')
           .with_environment(['HOME=/usr/share/foreman'])
           .with_logoutput('on_failure')
-          .with_refreshonly(true)
+          .with_refreshonly(false)
```

Comment:
This doesn't look correct because unless is not specified. In https://github.com/ekohl/puppet-foreman/pull/new/fixes I've written a test case for this.